sound/soc/sof/loader.c

```diff
@@ -803,7 +803,7 @@ int snd_sof_run_firmware(struct snd_sof_dev *sdev)
 	if (ret == 0) {
 		dev_err(sdev->dev, "error: firmware boot failure\n");
 		snd_sof_dsp_dbg_dump(sdev, SOF_DBG_REGS | SOF_DBG_MBOX |
```

Comment:
interesting, `snd_sof_dsp_dbg_dump()` calls `ops.dbg_dump()` which can be one of `hda_dsp_dump()`, `bdw_dump()`, `byt_dump()` or `imx8_dump()` and none of those used `flags` until now...